bin/plotting/pycbc_page_ifar_vs_stat

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+import sys, h5py
+import matplotlib
+matplotlib.use('agg')
+from matplotlib import pyplot as plt
+import numpy as np
+import argparse, logging
+from pycbc import init_logging, add_common_pycbc_options, results
+from pycbc.events import significance
+from pycbc.conversions import sec_to_year as convert_s_to_y
+
+plt.rcParams.update({
+    "text.usetex": True,
+})
+
+parser = argparse.ArgumentParser(usage='pycbc_page_ifar_vs_stat [--options]',
+                    description='Plots cumulative IFAR vs stat for'
+                          'backgrounds of different event types.')
+add_common_pycbc_options(parser)
+# get singles fit options
+significance.insert_significance_option_group(parser)
+parser.add_argument('--trigger-files', nargs='+',
+                    help='Paths to separate-event-type statmap files.')
+parser.add_argument('--ifo-combos', nargs='+',
+                    help='Which event types (detector combinations) to plot.')
+parser.add_argument('--output-file', required=True,
+                    help='Path to output plot.')
+parser.add_argument("--loudest-keep-values",
+                    default='[6:1]',
+                    help="Apply successive multiplicative levels of"
+                         " decimation to coincs with stat value below the"
+                         " given thresholds. Supply as a comma-separated list"
+                         " of threshold:decimation value pairs surrounded by"
+                         " square brackets (no spaces!). Decimation values must"
+                         " be positive integers."
+                         " Ex. [15:5,10:30,5:30,0:30]."
+                         " Default: no decimation")
+opts = parser.parse_args()
+init_logging(opts.verbose)
+
+sig_dict = significance.digest_significance_options(opts.ifo_combos, opts)
+
+coinc_color_map = {
+    'H':  '#ee0000',  # red
+    'L':  '#4ba6ff',  # blue
+    'V':  '#9b59b6',  # magenta/purple
+    'HL': '#0000DD',
+    'LV': '#FFCC33',
+    'HV': '#990000',
+    'HLV': np.array([20, 120, 20]) / 255.}
+
+coinc_line_map = {
+    'H':  '--',
+    'L':  '--',
+    'V':  '--',
+    'HL': '-',
+    'LV': '-',
+    'HV': '-',
+    'HLV':'-'
+}
+
+exc_zlag_filenames = []
+# Not sure what this step does precisely - select the first matching file ?
```

Comment:
This was for getting the files into the 'correct' order of the plotting for the singles paper

We can probably remove this and use opts.trigger_files directly